Tasks.md

```diff
@@ -1,94 +1,301 @@
-    - show the new score immediately, like we did in `startGame()`
-
-
-9) TIMEUP
-    - add a variable for whether the time is up or not
-        - which type?
-    - inside `peep()`, add the condition that you don't do anything if the time is up!
-    - when the game starts, set the variable to `false`
-    - after 15 seconds, finish the game by setting the variable to `true`
-        - tip: create a function `finish()` which sets the variable (to what?)
-        - before calling `finish()`, use the `sleep(time)` function
-        - before `sleep` write `await` and make the `startGame()` function `async`
-
-10) RESTART GAME
-    - the `setInterval` function actually returns an object, let's call it `clock` and assign it to a variable
-    - after finishing the game, call `clearInterval()` passing the `clock` as parameter
-    - add a log to the `peep()` function to see if it worked!
-    - now we're able to restart the game. Inside the `finish()` function, re-enable the start button
-
-### Extra exercises
+### 0 - HTML
+- In the HTML file, add the `"up"` class to the mole and watch it appear!
+    - hint: classes are separated by space, e.g. `class="oneClass anotherClass"`
+- Remove the `"up"` class and watch it (you guessed it) disappear
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+*HTML*
+```html
+<div class="mole up"></div>
+```
+</details>
+
+
+### 1) START GAME
+- create a function called `startGame`, which does nothing and has no parameters
+    - pay attention to uppercase letters!
+- disable the start button
+    - hint: the start button is already present in the Javascript file, it's called `startButton` and it's an object.
+    - hint: use the `disabled` property of the button, set it to `true`
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+```javascript
+// starts the game
+function startGame() {
+    startButton.disabled = true
+}
+```
+</details>
+
+
+### 2) PEEP
+- create a function `peep()` without parameters
+- create a `mole` variable inside this function
+- get the first element of the `moles` array and assign it to the `mole` variable
+    - remember, arrays start at `0`!
+- call `show` on that mole (i.e. pass that mole as parameter to the show function)
+- make the code wait 1 second
+    - hint: use the `sleep(time)` function where `time` is a number in milliseconds
+    - before `sleep` write `await`
+    - if the `peep` function includes an `await`, then it must be declared as "asynchronous". Just add `async` before the keyword `function`.
+- call `hide` on that mole after 1s has passed
+- call the `peep()` function from inside `startGame`
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+```javascript
+async function peep() {
+    const mole = moles[0]
+    show(mole)
+    await sleep(1000)
+    hide(mole)
+}
+
+function startGame() {
+    // ...
+    peep()
+}
```

Comment:
I think we should keep the same order as in `02.js`, so having `startGame` above `peep`